### PR TITLE
fix(oci): Set the command of the image

### PR DESCRIPTION
**Pull request by craciunoiuc**
### Prerequisite checklist

  - [x] Read the [contribution guidelines](https://unikraft.org/docs/contributing/) regarding submitting new changes to the project;
  - [x] Tested your changes against relevant architectures and platforms;
  - [x] Ran `make fmt` on your commit series before opening this PR;
  - [ ] Updated relevant documentation.

### CODE REVIEW

File: oci/pack.go [hunk 1]
```diff
@@ -441,6 +443,13 @@ unpack:
 		// Set the kernel, since it is a well-known within the destination path
 		ocipack.kernel = filepath.Join(popts.Workdir(), WellKnownKernelPath)
 
+		// Set the cmdline, since it is a well-known within the destination path
+		cmdPath := filepath.Join(popts.Workdir(), WellKnownCmdlinePath)
+
+		if err := os.WriteFile(cmdPath, []byte(cmd), 0644); err != nil {
```
Comment (antoineco) [inline oci/pack.go]:
```suggestion
		if err := os.WriteFile(cmdPath, []byte(cmd), 0o644); err != nil {
```

Comment (craciunoiuc) [reply oci/pack.go]: I want to see first if I can get/write this information without changes to kraftkit, let's see

Comment (nderjung) [reply oci/pack.go]: Please see #559. I think this supersedes this PR.

Comment (craciunoiuc) [reply oci/pack.go]: replaced

File: oci/pack.go [hunk 2]
```diff
@@ -441,6 +441,10 @@ unpack:
 		// Set the kernel, since it is a well-known within the destination path
 		ocipack.kernel = filepath.Join(popts.Workdir(), WellKnownKernelPath)
 
+		// Set the command
+		ocipack.command = image.Config.Cmd
+		ocipack.image.SetCmd(ctx, image.Config.Cmd)
```
Comment (nderjung) [inline oci/pack.go]: Populate the whole config structure:
```diff
- ocipack.image.SetCmd(ctx, image.Config.Cmd)
+ ocipack.image.config = image
```

**Review by nderjung — APPROVED** [review]

**Pull request merged**